### PR TITLE
add videoControlBarBgColor

### PR DESCRIPTION
**Pull request by nicktt19**
add videoControlBarBgColor. need to edit the control bar back ground colour. Please help. 
Thank you very much.

### CODE REVIEW

File: lib/embedded_tweet_view.dart [hunk 1]
```diff
@@ -182,25 +187,25 @@ class EmbeddedTweetView extends StatelessWidget {
...
-                            userScreenNameStyle:
-                                defaultQuoteUserScreenNameStyle,
-                            backgroundColor: null,
-                            borderColor: null,
-                            onTapImage: onTapImage,
-                            autoPlayVideo: autoPlayVideo,
-                            enableVideoFullscreen: enableVideoFullscreen,
-                          ),
-                        )
+                    padding: EdgeInsets.only(top: 8.0, bottom: 10),
```
Comment (wszeborowskimateusz) [inline lib/embedded_tweet_view.dart:190]: Looks like you reformatted te code - could you bring it back?

File: lib/src/tweet_video.dart [hunk 2]
```diff
@@ -4,20 +4,21 @@ import 'package:flutter/services.dart';
...
-    this.enableFullscreen = true,
-    this.videoHighQuality = true,
-  }) : super(key: key);
+  TweetVideo(this.tweetVM,
+      {Key? key,
+      this.initialVolume = 0.0,
+      this.autoPlay = false,
+      this.enableFullscreen = true,
+      this.videoHighQuality = true,
+      this.controlBarColor})
```
Comment (wszeborowskimateusz) [inline lib/src/tweet_video.dart:13]: please add a trailing comma

File: lib/src/tweet_video.dart [hunk 3]
```diff
@@ -44,6 +45,7 @@ class _TweetVideoState extends State<TweetVideo>
       },
       aspectRatio: widget.tweetVM.getDisplayTweet().videoAspectRatio!,
       controlsConfiguration: BetterPlayerControlsConfiguration(
+        controlBarColor: widget.controlBarColor??Colors.black.withOpacity(0.5),
```
Comment (wszeborowskimateusz) [inline lib/src/tweet_video.dart:48]: the code doesn't look formatted. We should have spaces inbetween

Comment (wszeborowskimateusz) [reply lib/src/tweet_video.dart:48]: Also as I see in the package the default color they use is `black87` I think we should use the same as default here,

File: lib/embedded_tweet_view.dart (hunk 1 above)
Comment (nicktt19) [reply lib/embedded_tweet_view.dart]: ok, i will do it right now. thanks for your comment

**Comment by wszeborowskimateusz** [comment]
Also please update CHANGELOG.md file

File: lib/src/tweet_video.dart [hunk 4]
```diff
@@ -4,20 +4,21 @@ import 'package:flutter/services.dart';
...
-    this.enableFullscreen = true,
-    this.videoHighQuality = true,
-  }) : super(key: key);
+  TweetVideo(this.tweetVM,
+      {Key? key,
+      this.initialVolume = 0.0,
+      this.autoPlay = false,
+      this.enableFullscreen = true,
+      this.videoHighQuality = true,
+      this.controlBarColor,})
```
Comment (wszeborowskimateusz) [inline lib/src/tweet_video.dart:13]: Also please format this file

**Comment by nicktt19** [comment]
ok no problem

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- Added videoControlBarBgColor for edit the video player control bar background
```
Comment (wszeborowskimateusz) [inline CHANGELOG.md:9]: Did you merge dev to your branch? I think it should have one more entry here

Comment (wszeborowskimateusz) [reply CHANGELOG.md:9]: yeah, there is a conflict

**Comment by wszeborowskimateusz** [comment]
ok, LGTM. I am merging it

**Comment by wszeborowskimateusz** [comment]
I will also release a new version with it

**Comment by nicktt19** [comment]
> ok, LGTM. I am merging it

Thank you so much

**Comment by wszeborowskimateusz** [comment]
@nicktt19 I published version 4.0.3

**Pull request merged**